src/com/kennymaness/Character.java

```diff
@@ -1,125 +1,129 @@
 package com.kennymaness;
 
 import java.util.ArrayList;
-import static com.kennymaness.Game.slowPrint;
 
-public class Character {
 
-    static final int maxHitPoints = 20;
+enum CharacterRace {
```

Comment:
Moved the enums outside of the class definitions into the outer scope of the files